nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+// still can't manage to get the meals with their reviews
+app.get("/meals", async (request, response) => {
+  response.json(meals);
+});
+
+app.get("/cheap-meals", (request, response) => {
+  cheapMeals = meals.filter((cheapmeal) => {
+    return cheapmeal.price < 50;
+  });
```

Comment:
This does the same. So you dont have to do this, but you can

```suggestion
  cheapMeals = meals.filter((cheapmeal) => cheapmeal.price < 50);
```